### PR TITLE
Command creation for grass from input

### PR DESCRIPTION
**Pull request by mayank33**

### CODE REVIEW

**Comment by mayank33** [comment]
screenshot from 2016-07-09 22 54 20

File: src/Overall.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#include <Wt/WHBoxLayout>
+#include <Wt/WImage>
+#include <Wt/WCheckBox>
+#include <Wt/WLineEdit>
+#include <Wt/WText>
+#include <Wt/WLabel>
+#include <Wt/WSelectionBox>
+
+
+#include "Text_Gui.h"
```
Comment (rashadkm) [inline src/Overall.h:26]: include only required headers

**Comment by rashadkm** [comment]
could you explain the class Text_Gui and Overall?

Also in the screenshot, there is the layout problem i mentioned on gitter.
the label and the text must come like in a horizontal layout. 

easy fix will be to set all tabs use horizontal layout. label, input widget (text, checkbox) will be on each row

## |Required|

## Name of input vector map

## <text box>

## Layer number or name ('-1' for all layers)

## <text box>

PS: bold text is a rendering by github. Not me shouting... ;)
Note that for now, there won't be any WSelectionBox inisde wgrass command module.
only textedit and checkbox

if the grass wxgui shows layername as a selectionbox, we change into simple text edit

in the bottom there on parameter called input feature type which three or four checkbox
so all the checkbox will go into a group or wcontainer widget and that goes into a single line below the label text 'Input feature type'

File: src/Overall.h (hunk 1 above)
Comment (mayank33) [inline src/Overall.h:26]: yeah, will do the code cleaning today.

**Comment by mayank33** [comment]
Text_Gui is a class which is made for the different options. I made that as every option hold a lot of data like name to shown in GUI, name for the grass command, type to be sent in grass command, whether it has multiple or not. These things will be used once data is entered and a grass command is to be made. Overall is combination of Text_gui and widget associated with it.  Initially I made all this for easyness and also I have to access all in Mainui class so just making an another class and acessing it in MainUi is easy.

**Pull request merged**